### DISCUSSION TITLE
PnP PowerShell ListsShowHeaderAndNavigation

### DISCUSSION THREAD

**Discussion by IfEyeOnlyKnew** [pnp/powershell#3227] [General] [answered]
**[SharePoint Online] Microsoft Lists User Experience Update [MC600726]**

Is there any work on having a parameter added to Set-PnPSite using -ListsShowHeaderAndNavigation. A request to make this parameter Tenant wide change for all Site Collections.

Set-SPOSite -Identity https://contoso.sharepoint.com/sites/site1 -ListsShowHeaderAndNavigation $true / $false

Set-PnPSite -Identity https://contoso.sharepoint.com/sites/site1 -ListsShowHeaderAndNavigation $true / $false

https://pnp.github.io/powershell/cmdlets/Set-PnPSite.html

**Comment by gautamdsheth** [comment, marked as answer]
It will be available in tomorrow's nightly builds and the next major versions whenever we release that.
